src/components/MaterialUI/componentProps.test.js

```diff
@@ -0,0 +1,63 @@
+import ComponentProps from "./componentProps";
+
+describe("Component Props", () => {
+	it("Sets the prop value", () => {
+		var componentProps = new ComponentProps();
```

Comment:
Can be a const....

Reply:
why const for a local variable? consts should be used for places where variable shouldn't be overwritten by design. In this case I don't care about it there aren't such requirements.

Reply:
Replaced all the var with let